scm/src/GFS_typedefs.F90

```diff
@@ -2380,15 +2408,18 @@ subroutine sfcprop_create (Sfcprop, IM, Model)
        end if
 
     end if
+       allocate (Sfcprop%rmol   (IM ))
```

Comment:
Reduce the indentation here as these lines are not in an `if` statement.

Reply:
I noticed this, but the same misindentation is in the FV3 version too (and brought over via Meld). Correcting it in the SCM introduces a difference that I'd rather not deal with. Perhaps if I remember, I can fix it in both places the next time we have cause to change GFS_typedefs.